src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -20,21 +35,28 @@
         <input
           data-cy="searchInput"
           type="text"
+          value={filters.query}
           className="input"
           placeholder="Search..."
+          onChange={event => {
+            dispatch(filterActions.addQuery(event.target.value));
```

Comment:
Ensure that `filterActions.addQuery` is the correct action for updating the search query. If the action is named differently in your Redux slice, update it accordingly.